src/voxxml/attributes.ts

```diff
@@ -4,6 +4,31 @@ type Input = 'dtmf' | 'speech' | 'dtmf speech'
 
 type Method = 'GET' | 'POST'
 
+type BankAccountType = 'consumer-checking' | 'consumer-savings' | 'commercial-checking'
+
+type PaymentMethod = 'credit-card' | 'ach-debit'
+
+type TokenType = 'one-time' | 'reusable' | 'payment-method'
+
+type Currency = 'usd'
+
+type PromptFor = 'payment-card-number'
+  | 'expiration-date'
+  | 'security-code'
+  | 'postal-code'
+  | 'bank-account-number'
+  | 'bank-routing-number'
+
+type PromptError = 'input-timeout'
```

Comment:
Should this just be `timeout`?

Reply:
I think it should be `input-timeout` since that will mirror our competitors.